docs/_Interface_IColony.md

```diff
@@ -1519,6 +1562,18 @@ Transfer some amount of obligated tokens. Can be called by the arbitration role.
 |_recipient|address|Recipient of the transferred tokens.
 
 
+### `uninstallExtension`
+
+Uninstall an extension from a colony. Secured function to authorised members.
```

Comment:
I'd add that this is not reversible. Even if you reinstall, it's a different instance of the extension. Perhaps also mention the intended flow for removing an extension (deprecating it, waiting for everything to complete, and then uninstalling it).